src/app/rosetta/lib/network.ml

```diff
@@ -258,28 +289,32 @@ module Status = struct
       module Mock = Impl (Result)
 
       let build ~best_chain_missing = {
-        Get_status.genesisBlock = {
-          stateHash = "GENESIS_HASH";
-          protocolState = {
-            consensusState = {
-              blockHeight = Unsigned.UInt32.one
+        Get_status_t.genesis_block_identifier = {
+            genesisBlock = {
+            stateHash = "GENESIS_HASH";
+            protocolState = {
+              consensusState = {
+                blockHeight = Unsigned.UInt32.one
+              };
             };
           };
         };
-        bestChain = if best_chain_missing then None
-          else Some [|{
-              stateHash = "STATE_HASH_TIP";
-              protocolState = {
-                blockchainState = {utcDate = Int64.of_int_exn 1_594_854_566};
-                consensusState = {blockHeight = Int64.of_int_exn 4 }
-              }
-            }|];
-        daemonStatus = {
-          peers = [|{peerId = "dev.o1test.net"}|]
-        };
-        syncStatus = `SYNCED;
-        initialPeers = [||];
-        networkID = "mina:testnet"
+        status = {
+          bestChain = if best_chain_missing then None
+            else Some [|{
+                stateHash = "STATE_HASH_TIP";
+                protocolState = {
+                  blockchainState = {utcDate = Int64.of_int_exn 1_594_854_566};
+                  consensusState = {blockHeight = Int64.of_int_exn 4 }
+                }
+                }|];
+          daemonStatus = {
+            peers = [|{peerId = "dev.o1test.net"}|]
+          };
+          syncStatus = `SYNCED;
+          initialPeers = [||];
+          networkID = "mina:testnet"
```

Comment:
Shouldn't this be computed dynamically like in other places?

Reply:
I didn't change this logic, just had to indent the code. But this is to build the test setup, using this mock values.